sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityTests.java

```diff
@@ -199,6 +337,38 @@ public void getToken() {
         verifyTokenNotEmpty(issuedToken);
     }
 
+    @Test
+    public void getTokenWithoutUser() {
+        // Arrange
+        CommunicationIdentityClientBuilder builder = createClientBuilder(httpClient);
+        client = setupClient(builder, "getTokenSync");
+        List<CommunicationTokenScope> scopes = Arrays.asList(CommunicationTokenScope.CHAT);
+
+        // Action & Assert
+        try {
+            client.getToken(null, scopes);
+        } catch (NullPointerException ex) {
+            return;
+        }
+        fail("An exception should have been thrown.");
```

Comment:
This is kinda hard to read. Can't we use `assertThrows`?

https://github.com/Azure/azure-sdk-for-java/search?q=assertthrows